cookiecutter.json

```diff
@@ -1,9 +1,11 @@
 {
   "app_name": "default-app-name",
   "variant": ["API only", "Web only", "Both"],
+  "css_addon": ["None", "Bootstrap"],
```

Comment:
IMO, None should be the last choice because we mostly use Bootstrap at Nimble. What do you think?
cc @hoangmirs

Reply:
Yeah, that makes sense

Reply:
ok, I will fix that! :)